src/entities/offerings.ts

```diff
@@ -82,10 +83,13 @@ export interface Price {
  */
 export interface PricingPhase {
   /**
-   * The duration of the purchase option price in ISO 8601 format.
-   * For applicable options (trials, initial/promotional prices), otherwise null
```

Comment:
This is no longer true. We changed the backend so the normal phases also included the period.